src/Xrefcheck/Config/Default.hs

```diff
@@ -56,6 +56,10 @@ networking:
   # a "429 Too Many Requests" response.
   maxRetries: 3
 
+  # Maximum number of links that can be followed in a single redirect
+  # chain.
+  maxRedirectFollows: 10
```

Comment:
This is a bit subjective concern: for me as a user, an immediate question would be: what happens if this number is exceeded? Do we skip the check, or something else? I would put a very brief clarification.